section2/exercises/if-statements.js

```diff
@@ -44,6 +44,13 @@ Log to the console the appropriate age range
 var dogAge = 3;
 // Write your conditional here
 
+if (dogAge <= "2"){
+  console.log("puppy");
+} else if (dogAge > "2"){
+  console.log("adult");
+} else if (dogAge >= "10"){
```

Comment:
🚨 here is another case where you can use the syntax of using `else` at the end for a catch all:
```js
if (<condition #1>) {
  behavior
} else if (<condition #2) {
  behavior
} else {
  behavior
}
```